recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
@jakirkham how do we feel about using 2 cpus in builds? I think it is going to lead to some undiscoverable bugs.
@jschueller This only takes 3 minutes to compile, why are you keen on keeping `-j2`?